src/sdk/namespace_fs.js

```diff
@@ -3119,6 +3149,9 @@ class NamespaceFS {
                             gpfs_options?.delete_version, bucket_tmp_dir_path);
                     }
                 }
+                if (is_dir_content) {
+                    await this._handle_latest_disabled_dir_content_xattr(fs_context, params.key, versioned_path, latest_ver_path);
```

Comment:
I know it's a function from the previous PR but when reading it again I can't understand intuitively what it's doing, so it means we need to rename it

Reply:
changed it. hope its better now